public/scss/_forms.scss

```diff
@@ -144,16 +144,18 @@ label:focus ~ .input-help-focused {
   opacity: 1;
 }
 
-.card-icon {
-    opacity: 0;
-    position: absolute;
-    right: 8px;
-    top: 50%;
-    margin-top: -13px;
-    z-index: 10;
-}
+form {
```

Comment:
All I really want in card-icon is this: https://github.com/mozilla/payments-ui/blob/0279c768a05411e6cdb0b8e3640436c5e732f292/public/scss/_cardicons-sprite.scss#L1, not all the rest.